packages/webpack-extraction-plugin/src/parseCSSRules.ts

```diff
@@ -37,7 +37,7 @@ export function parseCSSRules(css: string) {
 
     if (cssBucketName) {
       const cssRule = serialize([element], stringify);
-      const bucketEntry: CSSBucketEntry = element.type === MEDIA ? [cssRule, cssMeta!] : cssRule;
+      const bucketEntry: CSSBucketEntry = cssBucketName === 'm' ? [cssRule, cssMeta!] : cssRule;
```

Comment:
The GitHub actions warning is really hidden here. Can we consider to complain more visibly?

Reply:
Good call, did not even notice it. Now the rule will throw